package.json

```diff
@@ -36,34 +37,23 @@
     "@openzeppelin/contracts": "^3.1.0"
   },
   "devDependencies": {
-    "@crestproject/crestproject": "^0.0.23",
```

Comment:
I did a bunch of updates to dependencies here while debugging the test failure. Fixed the failure in crestproject but keeping these updates is not bad either way.